src/crepe/facade/SDLContext.h

```diff
@@ -1,48 +1,152 @@
 #pragma once
 
+#include <SDL2/SDL_keycode.h>
 #include <SDL2/SDL_render.h>
 #include <SDL2/SDL_video.h>
+#include <memory>
+#include <string>
 
 #include "../api/Sprite.h"
 #include "../api/Transform.h"
-#include "../system/RenderSystem.h"
+#include "api/Camera.h"
+
+//FIXME: this needs to be removed
+const int SCREEN_WIDTH = 640;
+const int SCREEN_HEIGHT = 480;
 
 namespace crepe {
+typedef SDL_Keycode CREPE_KEYCODES;
```

Comment:
Deze typedef is nog steeds niet bruikbaar vanuit de API omdat het onderliggende type door SDL2 gedefiniëerd is. Ik begrijp dat er tijdsnood achter deze PR zit (?) dus misschien is het handiger om het er als `// TODO: ` comment bij te zetten voor later omdat het nu geen directe problemen veroorzaakt